src/server/Demo.Web/.platform/nginx/conf.d/https.conf

```diff
@@ -20,6 +20,9 @@ server {
         proxy_set_header        X-Real-IP       $remote_addr;
         proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header        X-Forwarded-Proto https;
-        proxy_set_header        Access-Control-Allow-Origin *;
+        add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
Hmmm, is it even a good idea to have this stuff hard-coded?

Reply:
It can be limited for certain requests, or limit the origin to a certain location. Dunno if we want to be bothered, given there is no data at risk. Production grade cors policy will not have the wildcard in there, of course.

Reply:
Never mind, after some investigation and reading I believe I finally understand how this should work. :-)
The real server should only allow forwarded traffic from the proxy and the proxy is the one that knows the env. specific DNS name.